### PR TITLE
Document declarative configuration of Test Visibility in Jenkins using 'datadog' step

### PR DESCRIPTION
**Pull request by nikita-tkachenko-datadog**
### What does this PR do? What is the motivation?

Adds instructions for declaratively configuring Test Visibility in Jenkins using 'datadog' step.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 1]
```diff
@@ -541,10 +541,19 @@ This is an optional step that enables the collection of tests data using [Test V
 
 See the [Test Visibility documentation][17] for your language to make sure that the testing framework that you use is supported.
 
+There are different ways to enable Test Visibility inside a Jenkins job or pipeline:
+1. Using Jenkins configuration UI.
+2. Adding `datadog` step inside the pipeline script.
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:546]:
```suggestion
2. Adding the `datadog` step inside the pipeline script.
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 2]
```diff
@@ -541,10 +541,19 @@ This is an optional step that enables the collection of tests data using [Test V
 
 See the [Test Visibility documentation][17] for your language to make sure that the testing framework that you use is supported.
 
+There are different ways to enable Test Visibility inside a Jenkins job or pipeline:
+1. Using Jenkins configuration UI.
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:545]:
```suggestion
1. Using the Jenkins configuration UI.
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 3]
```diff
@@ -541,10 +541,19 @@ This is an optional step that enables the collection of tests data using [Test V
...
+2. Adding `datadog` step inside the pipeline script.
+3. Configuring the tracer manually.
+
 ### Enable with the Jenkins configuration UI
 
 UI-based Test Visibility configuration is available in Datadog Jenkins plugin v5.6.0 or newer.
 
+This option is not suitable for pipelines that are configured entirely in `Jenkinsfile` (for example, Multi-Branch pipelines or pipelines from Organization Folder).
+For these pipelines use declarative configuration with `datadog` step (described in the next section).
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:554]:
```suggestion
For these pipelines use declarative configuration with the `datadog` step (described in the next section).
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 4]
```diff
@@ -554,6 +563,47 @@ UI-based Test Visibility configuration is available in Datadog Jenkins plugin v5
...
+        stage('Example') {
+            steps {
+                echo "Hello world."
+            }
+        }
+    }
+}
+```
+
+In scripted pipeline, wrap the relevant section with the datadog step like so:
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:593]:
```suggestion
In a scripted pipeline, wrap the relevant section with the `datadog` step like so:
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 5]
```diff
@@ -554,6 +563,47 @@ UI-based Test Visibility configuration is available in Datadog Jenkins plugin v5
...
+datadog(testVisibility: [ enabled: true, serviceName: "my-service", languages: ["JAVASCRIPT"], additionalVariables: [:] ]) {
+  node {
+    stage('Example') {
+      echo "Hello world."
+    }
+  }
+}
+```
+
+The other datadog settings, such as `collectLogs` or `tags` can be added alongside the `testVisibility` block.
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:605]:
```suggestion
The other `datadog` settings, such as `collectLogs` or `tags` can be added alongside the `testVisibility` block.
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 6]
```diff
@@ -541,10 +541,19 @@ This is an optional step that enables the collection of tests data using [Test V
...
+There are different ways to enable Test Visibility inside a Jenkins job or pipeline:
+1. Using Jenkins configuration UI.
+2. Adding `datadog` step inside the pipeline script.
+3. Configuring the tracer manually.
+
 ### Enable with the Jenkins configuration UI
 
 UI-based Test Visibility configuration is available in Datadog Jenkins plugin v5.6.0 or newer.
 
+This option is not suitable for pipelines that are configured entirely in `Jenkinsfile` (for example, Multi-Branch pipelines or pipelines from Organization Folder).
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:553]:
```suggestion
This option is not suitable for pipelines that are configured entirely in `Jenkinsfile` (for example, Multibranch pipelines or pipelines from Organization Folder).
```

File: content/en/continuous_integration/pipelines/jenkins.md [hunk 7]
```diff
@@ -554,6 +563,47 @@ UI-based Test Visibility configuration is available in Datadog Jenkins plugin v5
 
 {{< img src="ci/ci-jenkins-plugin-tests-config.png" alt="Datadog Test Visibility configuration for Jenkins" style="width:100%;">}}
 
+### Enable with datadog pipeline step
```
Comment (jhgilbert) [inline content/en/continuous_integration/pipelines/jenkins.md:566]:
```suggestion
### Enable with the `datadog` pipeline step
```

**Review by jhgilbert — APPROVED** [review]
Approved with some grammatical suggestions, thanks!

**Review by drodriguezhdez — APPROVED** [review]

**Comment by nikita-tkachenko-datadog** [comment]
@jhgilbert could you please merge this? Corresponding Jenkins plugin feature has been released today

**Pull request merged**